### DISCUSSION TITLE
Schedule for regular release of ros2 humble on create 3?

### DISCUSSION THREAD

**Discussion by framlin** [iRobotEducation/create3_docs#261] [Questions] [answered]
Is there a concrete date planned, when the first regular version of ros 2 humble on create 3 will be released?

I have ordered a create 3 some days ago and wonder, if it is a good idea to start with an installation of H0.0.0 on it.
My PC runs ubuntu 22.04 with ros 2 humble and if it would only take a few weeks, I think it would be worth it to start directly with humble on the create 3, instead of installing galactic with docker or a VM on my PC.

**Comment by kwujciak** [comment, marked as answer]
For anyone with a similar question, the latest Humble release can be found [here](https://iroboteducation.github.io/create3_docs/releases/h_1_2/) (currently H.1.2).
